### DISCUSSION TITLE
JS-coded tutorial?

### DISCUSSION THREAD

**Discussion by usamabilal** [Drexel-UHC/SALURBAL-Heat-Study-MS85#6] [Ideas]
I was thinking that this may be a good paper to start building a tutorial that is not shiny-based on how to "look at the data". one of those infinite scroll pages with D3, @rl627 ?

**Comment by usamabilal** [comment]
could be something simple that @jlkephart writes in "slides", and then you translate into infinite scroll

**Comment by ran-codes** [comment]
## Thoughts/Context

The 'tutorial' that @usamabilal is talking about, I will refer to as 'Visual Essay'.

The shiny app allows for data exploration (we provide the audience with visualizations that allow them to drill down into the data very finely). While this may be of interest to data people or researchers it doesn't do a good job of telling a story. That's where the idea of a 'Visual Essay' aka long-form visualizations comes into place.  Here are some examples:

- From google's ML group: https://pair.withgoogle.com/explorables/hidden-bias/
- from a 'visual essay' news outlet: https://pudding.cool/2017/01/shape-of-slavery/
- undergraduate major story: https://cuthchow.github.io/college-majors-visualisation/

I think it's great to explore various mediums to tell stories. while so far at the UHC the primary medium is peer-reviewed papers and now manuscript supplemental apps, I think these types of visual essays are perhaps the most accessible to a general audience and have the highest chance of getting traction. 

## Next steps if we commit.

if @jlkephart  can provide the slides (text and what figures for each), I can provide the visual essay

**Reply by usamabilal** [reply]
Wow, these are great!! I think the google ML is the closest to what I was envisioning, but @jlkephart can decide on format.
On a broader scale, @rl627, I think we can try to use this as an example for future "visual essays". If the "slides" idea works as a beginning point, we could use @jlkephart (if we go this route) slides as an example for the future

**Comment by jlkephart** [comment]
This is great, I love it! I agree with story-telling as a framework, and I think that will translate well to other UHC projects. Personally, I like "visual story-telling" vs. visual essays, if we are going to try to market this approach across the UHC.

What format do you want this in Ran?

**Reply by ran-codes** [reply]
lets try ppt?

**Reply by ran-codes** [reply]
"visual story-telling" sounds more exciting! 😏
